modules/interacting-serverless-apps-http2-gRPC.adoc

```diff
@@ -12,28 +12,31 @@ To do this you must find the ingress gateway's public address and the applicatio
 .Procedure
 
 . Find the application host. See the instructions in _Verifying your serverless application deployment_.
-. The ingress gateway's public address can be determined using this command:
+. Get the ingress gateway's public address:
```

Comment:
It came up in chat recently to avoid using apostrophes in possessives (ISG). Maybe:
 
Get the public address of the ingress gateway: